app/frontend/src/components/UserSummary.tsx

```diff
@@ -19,7 +19,14 @@ export const useStyles = makeStyles((theme) => ({
     width: "100%",
   },
   title: {
-    marginBottom: theme.spacing(1),
+    marginTop: 0,
+  },
+  titleAndBarContainer: {
+    display: "flex",
+    flexDirection: "column",
+    justifyContent: "space-around",
```

Comment:
Re andrea's comment, I think to achieve what she is talking about, leave this as `space-around` but in the `root` class remove `alignItems`, which means it will go back to the default of `stretch` and the spacing will be balanced.

Reply:
That didn't make any difference to what I got now. I'm unsure if she meant what I got in the screenshot with the dev tool overlay or what I got now 🤔  @andreaowu ?